### ISSUE TITLE
fix bn problem

### ISSUE THREAD

**Issue by yqyao** [mahyarnajibi/SNIPER#87]
When I fix bn, the performance is so bad, I don't know why. @bharatsingh430 How about the your result using fix bn? In my experiment, batch-size is 8.

**Comment by bharatsingh430** [comment]
what’s the total batch size including all GPUs? If it’s just 8, drop learning rate by 10 and check once

**Comment by yqyao** [comment]
total batch size is 64. @bharatsingh430, 8 GPUs

**Comment by yqyao** [comment]
I drop learning rate by 10, but it don't work. @bharatsingh430 , RPN accuracy is only 73 after 5 epoch.

**Comment by bharatsingh430** [comment]
with 8 training with bn should be fine, still i can't say why accuracy is low

**Comment by yqyao** [comment]
If I don't use fp16 for training, which parameters in configs should I change? @bharatsingh430
